docs/2_registration_project_access_session.md

```diff
@@ -36,7 +37,7 @@ or
 dds user add [Email address] --role "Unit Personnel"
 ~~~
 
-# Create a project
+#### Create a project
```

Comment:
Can we move this and the changes below to another PR? Invitation / Registration should be in a separate section of the workshop imo from Project creation etc.

Reply:
You mean a new md file?

Reply:
No you could just remove these changes in this PR and add them to your other PR I think

Reply:
mmm ok

Reply:
Done, although now this PR just has two small things